.travis.yml

```diff
@@ -25,12 +28,24 @@ before_install:
   - ./runflake8
   - ./runisort
 
+  # Install NVM:
+  - curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.0/install.sh | bash
```

Comment:
we did a similar tasks in other modules and this was not needed, we want to keep the build step as simple and fast as possible, therefore I kindly ask to please remove it